packages/next/client/polyfills.js

```diff
@@ -1,3 +1 @@
-import '__next_polyfill__fetch'
-import 'url-polyfill'
-Object.assign = require('__next_polyfill__object_assign')
+import '@next/polyfill-nomodule'
```

Comment:
In the perfect world, we get rid of this. I'd prefer to **not** do that in this PR though so the review remains easy.